docs/documentation/accessibility.md

```diff
@@ -0,0 +1,136 @@
+# Accessibility
+
+This page goes over information relevant to both developers and users relating to accessibility considerations.
+
+Accessibility is very important to consider when developing applications.
+Colorblindness impacts roughly 4% of the general population, meaning that if your application gets even a small user base, there is a high probability that it is being used by someone with colorblindness.
+
+Fortunately, there are ways as both developers and users to address these accessibility concerns, as detailed on this page.
+
+## Colorblindness accessibility for users
+
+If you are a user of a **rich-click** CLI, there are a few options you have to improve accessibility for yourself.
+
+### 1. Configure your terminal's 4-bit ANSI colors
+
+The 4-bit ANSI color system is a set of 16 different colors implemented in every terminal. This is the most common way to set colors.
+These colors are **not** deterministic; different terminals use slightly different hex values for the ANSI colors. [Wikipedia has a full breakdown of all the variations in these colors](https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit)
+
+Most modern terminals have the ability to customize all of these colors in the terminals' settings.
+If you are having difficulty distinguishing colors, it is recommended that you adjust these settings.
+
+!!! note
+    This will only work for CLIs that utilize the 4-bit ANSI color system.
+    CLIs that utilize hex values or other color systems will not be impacted by your terminal's ANSI color settings.
+
+### 2. Use the `NO_COLOR` environment variable
```

Comment:
I think that this should probably come first. Then that mirrors the developer docs, it's also more likely to be useful for folks.